netplan_cli/cli/commands/apply.py

```diff
@@ -414,4 +414,5 @@ def process_ovs_cleanup(config_manager, ovs_old, ovs_current, exit_on_error=True
             if exit_on_error:
                 sys.exit(1)
         except OvsDbServerNotRunning as e:
-            logging.warning('Cannot call Open vSwitch: {}.'.format(e))
+            if utils.systemctl_is_active('ovsdb-server.service'):
```

Comment:
Hmm, `utils.systemctl_is_active()` is the method that is actually used to produce the `OvsDbServerNotRunning` error in the first place. So muting the warning message on that condition is doing unnecessary work, just to ignore it afterward.

I think we should rather consider implementing something like `utils.systemctl_list_units()` to check if that service unit is actually installed, and then skip all of `apply_ovs_cleanup()` (in `ovs.py`) in that case.

Additionally, we should make use of the `OPENVSWITCH_OVSDB_SERVER_UNIT` constant, instead of putting the `ovsdb-server.service` name directly.